### ISSUE TITLE
Lint wouso/games/quiz/models

### ISSUE THREAD

**Issue by palcu** [rosedu/wouso#787]
Lint:

```
models.py:15:80: E501 line too long (83 > 79 characters)
models.py:21:80: E501 line too long (91 > 79 characters)
models.py:29:80: E501 line too long (112 > 79 characters)
models.py:110:80: E501 line too long (88 > 79 characters)
models.py:120:80: E501 line too long (83 > 79 characters)
models.py:159:80: E501 line too long (90 > 79 characters)
models.py:165:80: E501 line too long (94 > 79 characters)
models.py:209:80: E501 line too long (81 > 79 characters)
models.py:241:80: E501 line too long (90 > 79 characters)
models.py:242:80: E501 line too long (96 > 79 characters)
models.py:243:80: E501 line too long (80 > 79 characters)
models.py:247:80: E501 line too long (86 > 79 characters)
models.py:285:80: E501 line too long (93 > 79 characters)
models.py:292:80: E501 line too long (88 > 79 characters)
models.py:299:80: E501 line too long (96 > 79 characters)
```

**Comment by raztechs** [comment]
Working on it

**Comment by raztechs** [comment]
Fixed in #794
